arcgis-ios-sdk-samples/Analysis/Statistical query (group and sort)/StatisticalQueryGroupAndSortViewController.swift

```diff
@@ -299,16 +299,16 @@ class StatisticalQueryGroupAndSortViewController: UIViewController, UITableViewD
                 selectedGroupByFieldNames.remove(at: indexPath.row)
                 
                 // Remove field from the order by fields
-                for (i, orderByField) in orderByFields.enumerated().reversed() {
+                for (index, orderByField) in orderByFields.enumerated().reversed() {
```

Comment:
These `for` loops should really just be replaced by a call to `remove(where:)`, but that can be done as part of a separate PR.

Reply:
Good call, I made the change since I would have to edit it either way.